qubesdbus/service.py

```diff
@@ -20,28 +20,37 @@
 # pylint: disable=invalid-name
 ''' Service classes '''
 
-from __future__ import absolute_import
-from typing import Any
-
+import asyncio
 import logging
+from typing import Any
 
 import dbus
 import dbus.mainloop.glib
 import dbus.service
-
+from dbus.service import BusName
 from systemd.journal import JournalHandler
 
-dbus.mainloop.glib.DBusGMainLoop(set_as_default=True)
+import gbulb
+from qubesadmin import Qubes
+from qubesadmin.events import EventsDispatcher
 
-from dbus.service import BusName
+gbulb.install()
+dbus.mainloop.glib.DBusGMainLoop(set_as_default=True)
```

Comment:
I don't think this is an issue, because you shouldn't use `service.py` outside of `qubesdbus`. It's there to save duplicate code. But if you insist I will change it.